src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public enum Color {
+    YELLOW {
+        @Override
+        public String toString() {
+            return "Yellow";
+        }

```

Comment:
You don't need to override toString method of enum values
```suggestion
    YELLOW
```